### PR TITLE
Add jobs for hive cmm branch.

### PR DESCRIPTION
**Pull request by abutcher**
/cc @2uasimojo

### CODE REVIEW

**Comment by 2uasimojo** [comment]
/retest

Not that it should matter, but https://github.com/openshift/hive/pull/1459 merged.

**Comment by 2uasimojo** [comment]
The e2e-azure failure is expected (for now).

/lgtm

**Review by hongkailiu — APPROVED** [review]

**Comment by 2uasimojo** [comment]
Ugh, something in the base metadata template must have changed since this succeeded yesterday. Will need to regenerate.

/hold

File: ci-operator/config/openshift/hive/openshift-hive-cmm.yaml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        name: hive
+      - env: RELEASE_IMAGE
+        name: release:latest
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: generic-claim
+- as: e2e-gcp
+  optional: true
```
Comment (2uasimojo) [inline ci-operator/config/openshift/hive/openshift-hive-cmm.yaml:172]: Note: We changed this to `run_if_changed: gcp` for master via #24483 / 9e44170516.

File: ci-operator/config/openshift/hive/openshift-hive-cmm.yaml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        name: hive
+      - env: RELEASE_IMAGE
+        name: release:latest
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-gcp
+- as: e2e-azure
+  optional: true
```
Comment (2uasimojo) [inline ci-operator/config/openshift/hive/openshift-hive-cmm.yaml:190]: Note: We changed this to `run_if_changed: azure` for master via #24483 / 9e44170516.

**Review by 2uasimojo — COMMENTED** [review]
You can update the `run_if_changed`s if ya like. Not a blocker.

**Comment by 2uasimojo** [comment]
/lgtm

Unhold when rehearsals pass.

**Comment by 2uasimojo** [comment]
/test pj-rehearse

Let’s at least see e2e pass

**Comment by 2uasimojo** [comment]
/test pj-rehearse

C'mon e2e, just one pass.

**Comment by 2uasimojo** [comment]
/hold

Needs generated config update.

**Comment by abutcher** [comment]
Bumped.

**Comment by 2uasimojo** [comment]
Still doesn’t look quite right

**Comment by abutcher** [comment]
Rebased, rebumped.

**Review by 2uasimojo — COMMENTED** [review]
/lgtm

**Pull request merged**